library/std/src/sys/unix/process/process_common.rs

```diff
@@ -296,6 +396,10 @@ impl Command {
     pub fn get_pgroup(&self) -> Option<pid_t> {
         self.pgroup
     }
+    #[allow(dead_code)]
+    pub fn get_signal_mask(&self) -> io::Result<&SignalSet> {
+        self.signal_mask.get_or_try_init(SignalSet::empty)
```

Comment:
We'll want to remove the OnceLock from this if the other PR's FCP doesn't get accepted.